src/learn/ApproveExperienceModal.js

```diff
@@ -12,112 +19,138 @@ import { I18n } from "@aws-amplify/core";
-            <Button className="btn-cancel" onClick={props.closeModal}>
+    <Dialog
+      fullWidth
+      open={show}
+      onClose={closeModal}
+      keepMounted
+      hideBackdrop={false}
+    >
+      <DialogTitle>
+        {I18n.get("reviewFor")} {activeExperience.title}
+        <IconButton
+          aria-label="close"
+          onClick={closeModal}
+          sx={{
+            position: "absolute",
+            right: 8,
+            top: 8,
+          }}
+        >
+          <MdClose />
+        </IconButton>
+      </DialogTitle>
+      <DialogContent className={classes.root}>
+        <h3>{I18n.get("description")}</h3>
+        <BlockContent blocks={activeExperience.overview} />
+        <h3>{I18n.get("notesForCoach")}</h3>
+        <p>{mongoExperience[activeExperience.priority].athleteNotes}</p>
+        <h3>{I18n.get("submitLink")}</h3>
+        <a
+          href={mongoExperience[activeExperience.priority].github}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {I18n.get("openLink")}
+        </a>
+        <br />
+        {mongoExperience[activeExperience.priority].coachNotes.length > 0 ? (
+          <>
+            <h3>{I18n.get("coachesNotes")}</h3>
+            <p>{mongoExperience[activeExperience.priority].coachNotes}</p>
+            <Button
+              size="large"
+              variant="contained"
+              color="primary"
+              onClick={closeModal}
+            >
               {I18n.get("close")}
             </Button>
-
-            {props.mongoExperience[props.activeExperience.priority].approved ===
-            true ? null : (
-              <>
-                {props.mongoExperience[props.activeExperience.priority]
-                  .revisionsNeeded === false ? (
+          </>
+        ) : (
+          <form onSubmit={handleSubmit(onSubmit)}>
+            <h3>{I18n.get("coachesNotes")}</h3>
+            <TextField
+              id="coachNotes"
+              variant="filled"
+              size="medium"
+              autoFocus
+              {...register("coachNotes")}
+            />
+            <DialogActions>
+              <Button
+                size="large"
+                variant="contained"
+                color="primary"
+                onClick={closeModal}
+              >
+                {I18n.get("close")}
+              </Button>
+              {mongoExperience[activeExperience.priority].approved ===
+              true ? null : (
+                <>
                   <Button
-                    className="btn"
-                    onClick={() => {
-                      props.markRequestRevisions(
-                        props.activeExperience,
-                        props.mongoExperience,
-                        coachNotes
-                      );
-                    }}
+                    size="large"
+                    variant="contained"
+                    color="primary"
+                    type="submit"
```

Comment:
This is not working sadly - the requestRevisions is not correctly reflected, meaning that when I, as a mentor, sign in to look at my mentees work and 'request revisions' - while, my local environment shows that they were sent for revisions, it's not the case.

What you need to do: can you create a new user, as a 'mentor' and then tell me what the email is? Then I will 'match you' as a mentor with them, and you will be able to test the interplay between the two.

Reply:
@mikhael28 ok, then could you please appoint fieldyslap@yandex.ru as a mentor and fieldyslap@inbox.ru as a mentee

Reply:
@mikhael28 Hi, Did you match users fieldyslap@yandex.ru as a mentor and fieldyslap@inbox.ru as a mentee? I checked yesterday both and I didn't see any changed. Maybe I do something wrong.

Reply:
Have you created those users already? I looked quickly in the directory to try and match them, but didn’t see them… I will take a look again! If you’d rather just use an existing generic test set of users, email me at jay@jayeclark.dev or hit me up on Discord and I’ll send you the credentials! (Will be at my PC, where I have the credentials info stored, in about an hour.)

Reply:
@jayeclark Yes, these users are already registered. I will email you. Thank you.

Reply:
@mikhael28 I've tested this issue locally on main and on paret0.com, and requestRevisions isn't 'working' on either of those. However, on all three (live site, local main, this PR branch) it's caused by an issue with the logic that governs how experience is displayed & updated, not with the actual updating of the database via the modal dialog. 

Basically: it's a valid bug but unrelated to @AleksandrKurilov's work converting the modals to MUI and the changes that need to be made are in files this PR doesn't touch.